x-pack/filebeat/module/panw/panos/ingest/pipeline.yml

```
@@ -134,24 +134,23 @@ processors:
  - set:
      field: network.direction
      value: inbound
-     if: 'ctx?.panw?.panos?.type == "TRAFFIC" && ctx?.panw?.panos?.source?.zone == "untrust" && ctx?.panw?.panos?.destination?.zone == "trust"'
+     if: 'ctx?.panw?.panos?.type == "TRAFFIC" && ctx?._temp_?.external_zones != null && ctx?._temp_?.internal_zones != null && ctx?.observer?.ingress?.zone != null && ctx?.observer?.egress?.zone != null && ctx._temp_.external_zones.contains(ctx.observer.ingress.zone) && ctx._temp_.internal_zones.contains(ctx.observer.egress.zone)'
```

That's a lot to take in, but it looks right. I found it easier to read like this since I can see all the logic at once.

```suggestion
     if: >
       ctx?.panw?.panos?.type == "TRAFFIC" && 
       ctx?._temp_?.external_zones != null && 
       ctx?._temp_?.internal_zones != null && 
       ctx?.observer?.ingress?.zone != null && 
       ctx?.observer?.egress?.zone != null && 
       ctx._temp_.external_zones.contains(ctx.observer.ingress.zone) && 
       ctx._temp_.internal_zones.contains(ctx.observer.egress.zone)
```

---

yeah those were awful, see if the new formatting is easier on the eyes.